domain/consensus/processes/dagtraversalmanager/dagtraversalmanager.go

```diff
@@ -145,3 +145,53 @@ func (dtm *dagTraversalManager) LowestChainBlockAboveOrEqualToBlueScore(highHash
 
 	return currentHash, nil
 }
+
+func (dtm *dagTraversalManager) CalculateSelectedParentChainChanges(
```

Comment:
DAG traversal has nothing to do with selected parent chain changes.
Consider renaming this to something more generic.